src/main/java/uk/gov/cshr/service/security/ClientDetails.java

```diff
@@ -20,6 +20,7 @@
 
     private static final Set<String> GRANT_TYPES = new HashSet<String>() {{
         add("implicit");
+        add("password");
```

Comment:
Note, I added this back in to make testing easier - much easier to request a password token than complete the implicit flow.